src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return function(a) {
+    return function(b) {
+      counter++;
+
+      if ((counter > 3 & counter % 2 === 0)) {
```

Comment:
Redundant parentheses
```suggestion
      if (counter > 3 & counter % 2 === 0) {
```